Solutions/JBoss/Data/Solution_Jboss.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "JBoss",
+  "Author": "Microsoft",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/contrastsecurity_logo.svg\"width=\"75px\"height=\"75px\">",
```

Comment:
@v-laanjana,change the logo file. It is redirecting to other logo.

Reply:
fixed